internal/config/config.go

```
@@ -0,0 +1,239 @@
+			logging.Logger.Infof("Loaded configuration from %s file", path)
+			loaded = true
+			break
+		}
+	}
+	
+	if !loaded {
+		logging.Logger.Infof("No .env file found or error loading it. Using environment variables or defaults.")
+	}
+	
+	// Map standard environment variables to our configuration keys
+	viperInstance.BindEnv("llm_api_key", "LLM_API_KEY", "OPENAI_API_KEY")
```

golangci-lint (1.64.8)

Error return value of `viperInstance.BindEnv` is not checked

(errcheck)